src/App.js

```diff
@@ -1,5 +1,69 @@
+import BaseballService from './service/BaseballService';
+
+import { InputView, OutputView } from './views';
+
+import { splitNumbersToArray } from './utils/splitNumbersToArray';
+
+import { MESSAGE } from './constants/message';
+import { RESTART_COMMAND } from './constants/system';
+import { ERROR_MESSAGE } from './constants/error';
+
+import CustomError from './exceptions/CustomError';
+
 class App {
-  async play() {}
+  #view = {
+    input: InputView,
+    output: OutputView,
+  };
+
+  #service = {
+    baseball: BaseballService.of(),
+  };
+
+  async play() {
+    await this.#processTurn();
+  }
+
+  async #processTurn() {
+    const userNumbers = splitNumbersToArray(
+      await this.#view.input.readLine(MESSAGE.enterSubmitBall),
+    );
+    const score = this.#service.baseball.computeScore(userNumbers);
+    this.#view.output.print(score);
+    await this.#checkCompleteGame();
```

Comment:
P4 - 개행을 통해서 더 가독성 있게 수정할 수 있을 것 같아요! 이번 공통 피드백 내용이기도 합니다 ㅎㅎ

```suggestion
    const userNumbers = splitNumbersToArray(
      await this.#view.input.readLine(MESSAGE.enterSubmitBall),
    );
    const score = this.#service.baseball.computeScore(userNumbers);
    
    this.#view.output.print(score);
    await this.#checkCompleteGame();
```